config/jail.conf

```diff
@@ -599,6 +599,16 @@ backend  = %(postfix_backend)s
 maxretry = 1
 
 
+[postfix-screendnbl]
```

Comment:
I'm still unsure it'd really expect a new jail.
I know there is one for RBL, but it is more for backwards compatibility reasons, however obsolete since mode rbl is included in mode normal too.

Reply:
intention was to implement an optional feature, not extending/overwriting existing definitions leading to by others unexpected behavior